modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/data/provider/test/DDMRESTDataProviderTest.java

```diff
@@ -615,6 +646,27 @@ private void _setUserPermissionChecker(boolean guest) throws Exception {
 			PermissionCheckerFactoryUtil.create(user));
 	}
 
+	private void _testGetDataWithLocale(
+			long ddmDataProviderId, Locale locale, String outputParameterId)
+		throws Exception {
+
+		DDMDataProviderRequest ddmDataProviderRequest =
+			_createDDMDataProviderRequest(
+				ddmDataProviderId, null, locale, null, null, null, null);
+
+		DDMDataProviderResponse ddmDataProviderResponse =
+			_ddmDataProvider.getData(ddmDataProviderRequest);
+
+		List<KeyValuePair> keyValuePairs = ddmDataProviderResponse.getOutput(
+			outputParameterId, List.class);
+
+		for (KeyValuePair keyValuePair : keyValuePairs) {
+			Assert.assertEquals(
+				LanguageUtil.get(locale, "country." + keyValuePair.getKey()),
+				keyValuePair.getValue());
```

Comment:
@p-albuquerque why do we have this `country.`?

Reply:
This is the pattern for the language keys that represent a country. [here](https://github.com/liferay/liferay-portal/blob/03dcce45704a24bf47a611a80ea766498b44aaca/modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties#L4438)